### PR TITLE
Minor spelling error corrections

### PR DESCRIPTION
**Pull request by DumbMachine**
Minor changes to correct spelling errors and remove unwanted spaces.

@shreyasbapat

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -56,12 +56,8 @@
 |circleci| |travisci| |appveyor| |codecov| |codeclimate|
 
 EinsteinPy is an open source pure Python package dedicated to problems arising in 
-General Relativity and relativistic physics, such as goedesics calculation for vacuum 
-solutions for Einstein's field equations, calculation of various quantities in these 
-geometries like Schwarzschild Radius and event horizon. The library also has functions 
-for Symbolic calculations in GR like Christoffel Symbols and much more is planned. 
-The library aims to solve Einstein's field equations for arbitarily complicated 
-matter distribution as one of the main goals. 
+General Relativity and relativistic physics, such as geodesics calculation for vacuum solutions for Einstein's field equations, calculation of various quantities in these geometries like Schwarzschild Radius and the event horizon. The library also has functions for Symbolic calculations in GR like Christoffel Symbols and much more is planned.
```
Comment (shreyasbapat) [inline README.rst]: We don't want to increase line length :/

Comment (DumbMachine) [reply README.rst]: I have made the changes to revert to the original line length.

**Comment by ritzvik** [comment]
@DumbMachine It takes efforts to find out these typos! Thanks!
Rebase(and if possible squash), then it's a go!

**Comment by ritzvik** [comment]
@shreyasbapat LGTM :) squash n merge this.

**Comment by shreyasbapat** [comment]
Aah! We don't follow commits regulation until too necessary. I am rebasing this.

**Pull request merged**